docs/index.md

```diff
@@ -1,8 +1,8 @@
-# playing-the-internship-game
+# Playing the Internship Game
 
-Content created by [Evy Kassirer](http://www.evykassirer.com/)
+Content created by [Evy Kassirer](http://www.evykassirer.com/) and [Rudi Chen](http://digitalfreepen.com/)
 
-Also viewable at [this website](https://evykassirer.github.io/playing-the-internship-game)
+You can view the source markdown files in the Docs folder. This content is also viewable at [this website](https://evykassirer.github.io/playing-the-internship-game)
```

Comment:
I think for now we could make them the same, since the overlap is quite large. It's reasonable to have the contributing section in the website too (if people land directly on the website, they can see that it's open for contribution). Instructions on how to build the websites aren't as useful on the website itself, but we could move it into a separate file (maybe even contributing.md).